### PR TITLE
Gyro conversion

### PR DESCRIPTION
**Pull request by tumbili**
With this the attitude estimator is working.

### CODE REVIEW

**Comment by tumbili** [comment]
@julianoes I need to fix the include

**Comment by julianoes** [comment]
@tumbili why is my commit in here as well, I think you need to rebase.

File: drivers/mpu9250/MPU9250.hpp [hunk 1]
```diff
@@ -34,11 +34,13 @@
 #pragma once
 
 #include "ImuSensor.hpp"
+#include <px4_defines.h>
 
 namespace DriverFramework {
 
 // update frequency 250 Hz
 #define MPU9250_MEASURE_INTERVAL_US 4000
+#define GYRO_RAW_TO_RAD_S 	32768.0f * 2000.0f * M_PI_2_F / 180.0f;		// 16 bit register, max 2000 degrees/s, deg to rad conversion
```
Comment (julianoes) [inline drivers/mpu9250/MPU9250.hpp]: The define should not have a semicolon at the and an expression like this should always be in brackets.

**Comment by tumbili** [comment]
@julianoes I did, don't know what's going on.

File: drivers/mpu9250/MPU9250.hpp [hunk 2]
```diff
@@ -34,12 +34,15 @@
 #pragma once
 
 #include "ImuSensor.hpp"
+//#include <px4_defines.h>
+
 
 namespace DriverFramework {
 
 // update frequency 250 Hz
+#define PI_HALF	1.57079632679489661923
```
Comment (tumbili) [inline drivers/mpu9250/MPU9250.hpp]: @julianoes I was having trouble getting this from the existing header files, could you have a look?

**Comment by tumbili** [comment]
https://www.youtube.com/watch?v=lBFBXRVrxs4&feature=youtu.be

**Comment by tumbili** [comment]
@julianoes @LorenzMeier Actuator controls topic is published correctly so if we get the pwm hack we should be close.

**Comment by julianoes** [comment]
@tumbili is this uncalibrated or calibrated? And is this including https://github.com/PX4/Firmware/pull/3960 ?

**Comment by tumbili** [comment]
@julianoes Uncalibrated, not including anything else!

**Comment by julianoes** [comment]
@tumbili interesting, ok :)

**Comment by julianoes** [comment]
@tumbili I think you got this wrong by a factor of two, not sure why you wanted PI/2, see #38.

**Comment by tumbili** [comment]
@julianoes Yeah, you are right!

**Pull request merged**